### PR TITLE
Update authentication service

### PR DESCRIPTION
**Pull request by jonandernovella**
Changes have been made in order to make it work with the new version of the service.

### CODE REVIEW

File: sda-svc/templates/auth-deploy.yaml [hunk 1]
```diff
@@ -53,7 +53,7 @@ spec:
 {{- end }}
         env:
         - name: LOG_LEVEL
-          value: {{ upper .Values.global.logLevel }}
+          value: {{ upper .Values.global.log }}
```
Comment (jbygdell) [inline sda-svc/templates/auth-deploy.yaml:56]:
```suggestion
          value: {{ upper .Values.global.logLevel }}
```
As per vaules file this should be `logLevel`

**Review by jbygdell — CHANGES_REQUESTED** [review]

File: sda-svc/templates/auth-deploy.yaml (hunk 1 above)
Comment (jonandernovella) [reply sda-svc/templates/auth-deploy.yaml:56]: https://github.com/neicnordic/sda-helm/blob/e8fa0498dd1eff9bafdc797d82728f5088798c92/.github/ci_tests/s3.yaml#L52

Comment (jbygdell) [reply sda-svc/templates/auth-deploy.yaml:56]: That explains why you don't see detailed logs in the tests.

This is how the templates look like:
```
      {{- if .Values.global.logLevel }}
        - name: LOG_LEVEL
          value: {{ .Values.global.logLevel | quote }}
      {{- end }}
```

Comment (jonandernovella) [reply sda-svc/templates/auth-deploy.yaml]: Fixed

File: sda-svc/templates/auth-deploy.yaml [hunk 2]
```diff
@@ -76,56 +76,32 @@ spec:
...
-        - name: KEY_FILE
+        - name: SERVER_KEY
           value: "{{ template "tlsPath" . }}/auth.key"
-        {{- if .Values.global.auth.cacert }}
-        - name: CA_CERTS
-          value: "{{ template "tlsPath" . }}/ca.crt"
-        {{- end }}
         {{- end }}
+        - name: S3INBOX
+          value: {{ .Values.global.inbox.s3Url }}
```
Comment (jbygdell) [inline sda-svc/templates/auth-deploy.yaml:101]: This should be *our* s3proxy, not the S3 backend.
Also the the ENV name should probably be `S3INBOX_URL` but that will require changed to the auth service.
```suggestion
        - name: S3INBOX
          value: {{ .Values.global.ingress.hostName.s3Inbox }}
```

**Review by jbygdell — CHANGES_REQUESTED** [review]

**Review by pontus — APPROVED** [review]
I wonder why all environment variables have been slightly renamed, but don't spot anything obviously wrong.

**Comment by blankdots** [comment]
LGTM, but are you going to change `release-v0.5.0` to  `latest` before merge ?

**Comment by jonandernovella** [comment]
> LGTM, but are you going to change `release-v0.5.0` to `latest` before merge ?



> LGTM, but are you going to change `release-v0.5.0` to `latest` before merge ?

Done!

**Review by jbygdell — APPROVED** [review]

**Pull request merged**